Jenkinsfile

```diff
@@ -23,19 +23,13 @@ pipeline {
       }
     }
     stage('deploy') {
-      agent {
-        dockerfile {
-          dir 'jenkins'
-          label 'docker-4x'
```

Comment:
really? any particular reason it was using `docker-4x`?

Reply:
`docker-push` make target runs `docker-build` target first, so it's ok!

it uses docker-4x because the rest of the stages used it (the build stages build a pile of images with docker-compose in parallel, which allows them to complete in a reasonable amount of time). There was not real reason the deploy was using 4x though, so i removed it :)